_docs/reference/notes/0.3.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Istio 0.3
+
+order: 50
+
+layout: docs
+type: markdown
+---
+
+## General
+
+Starting with 0.3, Istio is switching to a monthly release cadence. We hope this will help accelerate our ability
+to deliver timely improvements. See [here](./feature-stages.html) for information on the state of individual features
+for this release.
+
+This is a fairly modest release in terms of new features as the team put emphasis on internal
+infrastructure work to improve our velocity. Many bugs and smaller issues have been addressed and
+overall performance has been improved in a number of areas.
+
+## Security
+
+- **Secure Control Plane Communication**. Mixer and Pilot are now secured with mutual TLS, just like all services in a mesh.
+
+- **Selective Authentication**. You can now control authentication on a per-service basis via service annotations,
+which helps with incremental migration to Istio. [TBD: DOC LINK?]
+
+## Networking
+
+- **Egress RUles for TCP
```

Comment:
typo ^ maybe you meant:
- **Egress Rules for TCP**

Reply:
Whoops, I hadn't finished typing the whole thing in. Sorry about that.

Fixed.